db_test.go

```diff
@@ -743,28 +743,56 @@ func TestWALFlushedOnDBClose(t *testing.T) {
 }
 
 func TestWALSegmentSizeOption(t *testing.T) {
-	options := *DefaultOptions
-	options.WALSegmentSize = 2 * 32 * 1024
-	db, delete := openTestDB(t, &options)
-	defer delete()
-	app := db.Appender()
-	for i := int64(0); i < 155; i++ {
-		_, err := app.Add(labels.Labels{labels.Label{Name: "wal", Value: "size"}}, i, rand.Float64())
-		testutil.Ok(t, err)
-		testutil.Ok(t, app.Commit())
+	tests := map[int]func(dbdir string, options Options){
+		// Test Default Wal Size
+		0: func(dbDir string, options Options) {
+			files, err := ioutil.ReadDir(filepath.Join(dbDir, "wal"))
+			testutil.Ok(t, err)
+			for i, f := range files {
+				if len(files)-1 != i {
+					testutil.Equals(t, int64(DefaultOptions.WALSegmentSize), f.Size(), "WAL file size doesn't match WALSegmentSize option, filename: %v", f.Name())
+					continue
+				}
+				testutil.Assert(t, int64(DefaultOptions.WALSegmentSize) > f.Size(), "last WAL file size is not smaller than the WALSegmentSize option, filename: %v", f.Name())
+			}
+		},
+		// Test Custom Wal Size
```

Comment:
full stops for all comments and no need to add `Test` for all comments.
```suggestion
		// Custom Wal Size.
```